modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm

```diff
@@ -2,6 +2,178 @@
 ///CODE FOR LEWD QUIRKS///
 //////////////////////////
 
+/////////////////
+///BIMBO TRAIT///
+/////////////////
+
+/datum/brain_trauma
+	///Whether the trauma will be displayed on a scanner or kiosk
+	var/display_scanner = TRUE
+
+/datum/brain_trauma/special/bimbo
+	name = "Permanent hormonal disruption"
+	desc = "The patient has completely lost the ability to form speech and seems extremely aroused."
+	scan_desc = "permanent hormonal disruption"
+	gain_text = span_purple("Your thoughts get cloudy, but it turns you on like hell.")
+	lose_text = span_warning("A pleasant coolness spreads throughout your body, You are thinking clearly again.")
+	//people need to be able to gain it through the chemical OD
+	can_gain = TRUE
+	//people should not be able to randomly get this trauma
+	random_gain = FALSE
+	//we don't want this to be displayed on a scanner
+	display_scanner = FALSE
+	resilience = TRAUMA_RESILIENCE_LOBOTOMY
+	///how satisfied the person is, gained through climaxing
```

Comment:
Do code doc comments work properly when the line below is a regular comment?

Reply:
> Do code doc comments work properly when the line below is a regular comment?

In VSC, at least, yes.